Documentation/contributing/development/contributing_guide.rst

```diff
@@ -267,8 +267,11 @@ Pull requests review process for committers
 #. Once a PR is open, GitHub will automatically pick which `teams <https://github.com/orgs/cilium/teams/team/teams>`_
    should review the PR using the ``CODEOWNERS`` file. Each committer can see
    the PRs they need to review by filtering by reviews requested.
-   A good filter is provided in this `link <https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+review-requested%3A%40me+sort%3Aupdated-asc>`_
-   so make sure to bookmark it.
+   A good filter is provided in this `link <https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+assignee%3A%40me+sort%3Aupdated-asc>`_
+   so make sure to bookmark it. If a review is assigned to one or more
+   committers, it is expected those committers to review that PR. Once they have
+   performed their review, they should remove themselves from the list of
+   assignees.
```

Comment:
The documentation test is failing because we don't have that word in the dictionary yet.